### ISSUE TITLE
Speed plotter

### ISSUE THREAD

**Issue by Krokodealer** [guydavis/machinaris#91]
Hello! Can you test and rate the profitability of implementing that plotter into machinaris? 
https://github.com/madMAx43v3r/chia-plotter

**Comment by guydavis** [comment]
Hi, I think you meant to post a different link? Congratulations on the certificate.  Is it [this one](https://github.com/madMAx43v3r/chia-plotter)?  What were your findings with regard to:

1. Plotting speed (stock vs this plotter)
2. Plotting stability (any crashes or hangs?)
3. Plotting validity (`chia plots check` etc)?

**Comment by Krokodealer** [comment]
> Hi, I think you meant to post a different link? Congratulations on the certificate. Is it [this one](https://github.com/madMAx43v3r/chia-plotter)? What were your findings with regard to:
> 
> 1. Plotting speed (stock vs this plotter)
> 2. Plotting stability (any crashes or hangs?)
> 3. Plotting validity (`chia plots check` etc)?

My bad, fixed link.

**Comment by Krokodealer** [comment]
> Hi, I think you meant to post a different link? Congratulations on the certificate. Is it [this one](https://github.com/madMAx43v3r/chia-plotter)? What were your findings with regard to:
> 
> 1. Plotting speed (stock vs this plotter)
> 2. Plotting stability (any crashes or hangs?)
> 3. Plotting validity (`chia plots check` etc)?

1. Solo plot in stock plotter is done in about 4.5 hours, with this plotter I get solo plot per 25-35 minutes
2. No crashes, very stable.
3. All plots valid, already created about ~30 plots with this plotter.

**Comment by iamchriswick** [comment]
I'm running Plotter-only Machinaris with 40 plots simultaneously with a 10 min stagger. Plots are done in around 6 hours, and fully archived in 06:15. Currently running 5 x this setup.

**Comment by guydavis** [comment]
License for this project is [GPL v3.0](https://github.com/madMAx43v3r/chia-plotter/blob/master/LICENSE)

**Comment by Krokodealer** [comment]
> License for this project is [GPL v3.0](https://github.com/madMAx43v3r/chia-plotter/blob/master/LICENSE)

Something wrong with it?

**Comment by ibobo** [comment]
Yes please, this would be awesome!

**Comment by guydavis** [comment]
https://github.com/Chia-Network/chia-blockchain/discussions/452#discussioncomment-854926

**Comment by rangulicon** [comment]
JM from Chia talking about this plotter here: https://www.youtube.com/watch?v=PCViU6vhdcs

**Comment by rangulicon** [comment]
> [Chia-Network/chia-blockchain#452 (reply in thread)](https://github.com/Chia-Network/chia-blockchain/discussions/452#discussioncomment-854926)

That conversation is pretty interesting. Glad to see constructive things happening and that they have committed to ensuring the improvements are more broadly available.

**Comment by JCandle** [comment]
hi @guydavis, do you have plans to implement this? I love your GUI on unraid because of the simplicity. Doing this all via CLI by myself sounds daunting.

**Comment by guydavis** [comment]
Yes!  Check out initial MadMax plotter support in the test version of Machinaris.  More details on the [integration in the Wiki](https://github.com/guydavis/machinaris/wiki/MadMax).

**Comment by guydavis** [comment]
Big thanks to @Krokodealer for the tip on this.  Sorry for taking so long, but I spent a week allowing Machinaris to [manage multiple workers](https://github.com/guydavis/machinaris/wiki/Workers) across your LAN.  Once I got time for Madmax, I found that adding support to Plotman (Machinaris' plotting manager) was fairly easy.

As [outlined here](https://github.com/guydavis/machinaris/wiki/Workers), to start with, set Plotman to a single global job and modify the threads & buckets variables to suit your system.
